external/include/mbedtls/config.h

```diff
@@ -2701,9 +2701,9 @@
  *
  * This module is required for X.509 CRL parsing.
  */
-#if defined(MBEDTLS_OCF_PATCH)
```

Comment:
This modification in public repo might incur a potential problem for other applications and/or security framework. Or I think X509 option is independent of OCF Patch. Please verify that the source code related width MBEDTLS_X509_CRL_PARSE_C is independent on MBEDTLS_OCF_PATCH.